include/install-vitasdk.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+
+get_download_link () {
+  curl "https://api.github.com/repos/vitasdk/autobuilds/releases" | grep "master" | grep "browser_download_url" | grep $1 | head -n 1 | cut -d '"' -f 4
```

Comment:
GitHub API's keeps being rate-limited from Travis, use this instead: https://github.com/vitasdk/vita-headers/blob/master/.travis.d/download_sdk.sh

Reply:
this would add a dep on python